crates/scarb-api/src/version.rs

```diff
@@ -0,0 +1,42 @@
+use crate::ScarbCommand;
+use anyhow::{Context, Result};
+use regex::Regex;
+use semver::Version;
+use std::str::from_utf8;
+
+pub struct ScarbVersionOutput {
+    pub scarb: Version,
+    pub cairo: Version,
+}
+
+pub fn scarb_version_command() -> Result<ScarbVersionOutput> {
```

Comment:
Can we add a follow-up issue to refactor this, after the `Metadata` is integrated into the crate as well? 
Namely: i think this should be accessible via the `ScarbCommand` struct from the crate (through a new constructor method or something like that)

Reply:
I'm not sure because we already can get `scarb --version` output from the command. It's just a utility function and we are not providing any in the command itself

Reply:
But we probably will soon enough